govstack_api/building_blocks/bb_digital_registries/registries/insuree_registry.py

```diff
@@ -0,0 +1,209 @@
+import base64
+import json
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+
+from django.apps import apps
+from govstack_api.building_blocks.bb_digital_registries.registries.base_registry import BaseRegistry
+from govstack_api.graphql_api_client import GrapheneClient
+from insuree.schema import Query, Mutation
+
+
+config = apps.get_app_config('govstack_api')
+
+
+def get_insurees_query(arguments_with_variables: str = "", fetched_fields: str = "") -> str:
+    return f'''
+            query GetInsurees {{
+                insurees({arguments_with_variables}) {{
+                    edges{{
+                        node{{
+                            {fetched_fields}
+                        }}
+                    }}
+                }}
+            }}
+            '''
+
+
+def get_update_registry_query(variables_values: str = "") -> str:
+    query = f'''
+                mutation {{
+                  updateInsuree(
+                    input: {{
+                      clientMutationId: "552f8e55-ed5a-4e1e-a159-ea8f8cec0560"
+                      clientMutationLabel: "Update insuree"
+                      {variables_values}
+                    }}
+                  ) {{
+                    clientMutationId
+                    internalId
+                  }}
+                }}
+                '''
+    return query
+
+
+
+def get_create_registry_query(
+        arguments_with_values: str = "",
+        family_id="familyId: 1",
+        gender_id='genderId: "U"',
+        dob=config.date_of_birth,
+        head='head: false',
+        card_issued='cardIssued: false'
+) -> str:
+    return f'''
+        mutation {{
+          createInsuree(
+            input: {{
+              clientMutationId: "9d738f06-7855-455a-840f-99aa85d2308e"
+              clientMutationLabel: "Create insuree"
+              chfID: "{arguments_with_values['id']}"
+              {arguments_with_values}
+              {family_id}
+              {gender_id}
+              {dob}
+              {head}
+              {card_issued}
+            }}
+          ) {{
+            clientMutationId
+            internalId
+          }}
+        }}
+        '''
+
+
+class InsureeRegistry(BaseRegistry):
```

Comment:
While I like new architecture for this module, the InsureeRegistry still has multiple hardcoded parts like fields mapping and gql query templates. Finally we need to have possibility to define registries based on the minimal setup and ideally no coding, only configuration.